desktop/core/src/desktop/js/reactComponents/PathBrowser/PathBrowser.tsx

```diff
@@ -133,7 +135,7 @@ const PathBrowser = ({
                       className="hue-path-browser__breadcrumb-seperator"
                       data-testid={`${testId}-breadcrumb-seperator`}
                     >
-                      {seperator}
+                      {index === 0 && fileSystem !== 'hdfs' ? '://' : seperator}
```

Comment:
We should spell this **separator**

and for clarity we could introduce a variable that signals that this is not the normal separator
`URISchemeSeparator = fileSystem == 'hdfs' ? seperator : '://';`